addons/hw_drivers/tools/helpers.py

```diff
@@ -273,11 +273,27 @@ def get_odoo_server_url():
 def get_token():
     return read_file_first_line('token')
 
-def get_version():
+
+def get_commit_hash():
```

Comment:
in my opinion it is better to use a dedicated appropriate command rather than reusing something which is not really meant for that
+ it will trigger security ping anyway as otherwise I will have to move the git command elsewhere